phase1/gce/gce.jsonnet

```diff
@@ -185,6 +186,7 @@ function(cfg)
             "k8s-kubeadm-master-ip": "${google_compute_instance.%(master_instance)s.network_interface.0.address}" % names,
           } + if p2.provider == "kubeadm" then {
             "k8s-kubeadm-token": "${var.kubeadm_token}",
+            "kube-proxy-mode": "%(proxy_mode)s" % p2,
```

Comment:
Nitpick: This shouldn't be needed on the node template, right? It looks like only the master template uses it, so this will just get ignored here.

Reply:
Yes, my mistake, I have fixed that, thanks.